content/gui/xml/ingame/widgets/captains_log.xml

```diff
@@ -22,10 +23,12 @@
 	</VBox> <!-- closes VBox logbook left page -->
 
 	<!-- right page -->
-	<VBox max_size="350,508" min_size="350,508" padding="5" name="logbook_right">
-		<VBox name="custom_widgets_right" max_size="350,508" min_size="350,0"
-			comment="In here, widgets defined in yaml get added on runtime." />
-	</VBox> <!-- closes VBox logbook right page -->
+    <VBox max_size="350,508" min_size="350,508" padding="5" name="logbook_right">
```

Comment:
Looks like this is not indented the right way. The line starts earlier than the comment above it.